purview/models/bugzilla.py

```diff
@@ -11,16 +11,19 @@ class BugzillaBug(StructuredNode):
     assignees = RelationshipTo('.user.User', 'ASSIGNED_TO')
     reporters = RelationshipTo('.user.User', 'REPORTED_BY')
     qa_contacts = RelationshipTo('.user.User', 'QA_BY')
-    severity = StringProperty(required=True)
-    status = StringProperty(required=True)
-    creation_time = DateTimeProperty(required=True)
-    modified_time = DateTimeProperty(required=True)  # delta_ts
-    priority = StringProperty(required=True)
+    related_by_commits = RelationshipTo('.distgit.DistGitCommit', 'RELATES_TO')
```

Comment:
I was wondering if the relationship name for this should be in past tense too..something like "RELATED_TO"

Reply:
I'm torn on this myself since everything else is past-tense but technically, we are recording things that are related to each other, so I think `RELATES_TO` is correct since they are currently relating to each other. @ralphbean is the expert on grammar. Care to weigh in here? :stuck_out_tongue:

Reply:
The difference here is not actually past/present tense - but is active/passive voice.

- The bug relates to the commit.
- The bug is related to the commit.

These are both present tense, but the first is active voice and the second is passive (in the second the bug isn't "doing" the relating, it is more like a passive victim of the relationship).

Differences of voice only really make a difference in prose.  Either is fine here.  If you need a decision, I guess use "RELATED_TO" only because the "*ED_*" string will be easier to remember with all the other "*ED_*" strings floating around.

Reply:
Okay, going with "ED" for consistency with the other relationships.